ui/apps/platform/src/Containers/MainPage/Banners/CredentialExpiryBanner.tsx

```diff
@@ -52,7 +52,7 @@ function CredentialExpiryBanner({
     const message = (
         <span className="flex-1 text-center">
             {`${name} certificate ${getCredentialExpiryPhrase(expirationDate, now)}. `}
-            {hasAdministrationWritePermission ? (
+            {showCertGenerateAction ? (
                 <>To use renewed certificates, {downloadLink} and apply it to your cluster.</>
             ) : (
                 'Contact your administrator.'
```

Comment:
Is this accurate in the case of managed central? Who is the "administrator" in this case?

Reply:
I see your point, but because we don't expose whether we are running in ACSCS or not (we only exposed "Capabilities"), there still is no way for the UI to query the API about this.

If we can convince the folks who resist exposed that information through the API, then the API can vary the language used based on that. Until then, we have this.